test/e2e/manifests/meteringconfigs/mysql.yaml

```diff
@@ -42,6 +42,9 @@ spec:
 
   hive:
     spec:
+      image:
+        repository: quay.io/tflannag/hive
+        tag: db
```

Comment:
This image contains a slightly more up-to-date postgres-jdbc jar (42.2.3 vs. 4.2.14) and adds/removes the mysql-connector-java jar in favor of the mariadb one we started using in the 4.6 images.